mlrun/model_monitoring/features_drift_table.py

```diff
@@ -93,7 +93,6 @@ def __init__(self):
 
     def produce(
         self,
-        features: list[str],
```

Comment:
remove `features` from the docstring as well

Reply:
a79ba82